cli/lib/nf3.mjs

```diff
@@ -563,13 +563,33 @@ class Nf3 {
     const res = await axios.post(`${this.optimistBaseUrl}/proposer/register`, {
       address: this.ethereumAddress,
     });
+    if (res.data.txDataToSign === '') return false; // already registered
     return this.submitTransaction(
       res.data.txDataToSign,
       this.proposersContractAddress,
       this.PROPOSER_BOND,
     );
   }
 
+  /**
+  Registers a proposer locally with the Optimist instance only.  This will cause
+  Optimist to make blocks when this proposer is current but these will revert if
+  the proposer isn't registered on the blockchain too.  This method is useful only
+  if the proposer is already registered on the blockchain (has paid their bond) and
+  for some reason the Optimist instance does not know about them, e.g. a new instance
+  has been created. The method 'registerProposer' will both register the proposer
+  with the blockchain and register locally with the optimist instance. So, if
+  that method has been used successfully, there is no need to also call this method
+  @method
+  @async
+  @returns {Promise} A promise that resolves to the Ethereum transaction receipt.
+  */
+  async registerProposerLocally() {
+    return axios.post(`${this.optimistBaseUrl}/proposer/registerlocally`, {
```

Comment:
Proposer registration has changed so that now it includes a URL. Does `registerLocally` need to pass the URL as well? Probably not needed as this info is stored in the smart contract

Reply:
This endpoint is removed in the latest code push